pom.xml

```diff
@@ -35,7 +42,7 @@
 		<warContext>basicauth</warContext>
 		<package.file>${project.build.directory}/${project.artifactId}.zip</package.file>
```

Comment:
You can remove line 43 and 52 and 122 since package-server goal defaults to `${project.build.directory}/${project.artifactId}.zip` for `usr` type package and `${project.build.directory}/${project.artifactId}.jar` for `runnable` package type